compiler/rustc_expand/src/expand.rs

```diff
@@ -1852,6 +1843,57 @@ fn build_single_delegations<'a, Node: InvocationCollectorNode>(
     })
 }
 
+/// Required for `visit_node` obtained an owned `Node` from `&mut Node`.
+trait DummyValue {
```

Comment:
```suggestion
trait DummyAstNode {
```
Could you keep the old name `DummyAstNode`?
It was quite clear and specific, and consistent with `InvocationCollectorNode`.